### DISCUSSION TITLE
Calendar view for daily notes?

### DISCUSSION THREAD

**Discussion by dzintars** [zk-org/zk-nvim#196] [General]
Is there some tricks/plugins to get some kind of calendar view for `./zk/path/daily/*.md` notes?
I would like to see the general overview of my daily note calendar. Ideally... by picking the date, i would love to open that note or to create a new one. By hovering over... i would love to see preview.

I can see, how it could look in the Telescope:

![image](https://github.com/user-attachments/assets/91cd1e12-a926-4ada-92ba-b017665db368)

This image collage is taken from [Telekasten](https://github.com/nvim-telekasten/telekasten.nvim)

It could be of course more fancier, with the search and who knows what else... but general idea stays.

**Comment by tjex** [comment]
Ooft wow. Didn't know Telekasten had this. Really nice... 😏
I don't have any tricks personally.

But off the top of my head, if there is some neovim plugin that shows a date picker based on dates, then you could use the [`zk list` api](https://github.com/zk-org/zk-nvim#api) to feed whichever plugin the dates and paths from your diary notes. And open the path with `vim.cmd(e <path>)` (Not 100% if that's exctly the lua/vim cmd for it, but just to communicate the idea.

Please report back if you get anything going!
